metrics-server.tf

```diff
@@ -59,7 +59,54 @@ resource "helm_release" "metrics-server" {
     local.values_metrics-server,
     local.metrics-server["extra_values"]
   ]
+
+  #TODO(bogdando): create a shared template and refer it in addons (copy-pasta until then)
```

Comment:
NOTE: these are left in code for all the addons that currently support the ECR images prepare feature, to make sure we can address that, one day